__tests__/endpoints.test.js

```diff
@@ -52,3 +52,40 @@ describe("GET /api", () => {
       });
   });
 });
+
+// GET /api/articles/:article_id
+describe('GET /api/articles', () => { 
+  test('should return article with a specified id', () => { 
+    return query(app)
+      .get('/api/articles/1')
+      .expect(200)
+      .then((res) => {
+          const article = res.body;
+          expect(typeof article.article_id).toBe('number');
+          expect(typeof article.topic).toBe('string');
+          expect(typeof article.author).toBe('string');
+          expect(typeof article.body).toBe('string');
+          expect(typeof article.created_at).toBe('string');
+          expect(typeof article.votes).toBe('number');
+          expect(typeof article.article_img_url).toBe('string');
+      });
+  });
+  test('should return 404 if no articles found with provided id ', () => { 
+    return query(app)
+      .get('/api/articles/10000')
+      .expect(404)
+      .then((res) => {
+          const {errorMessage} = res.body;
+          expect(errorMessage).toBe('Not found');
+      });
+  });
+  test.only('should return 400 if provided id is not a number ', () => { 
```

Comment:
Make sure you're removing .only's I can't be sure any of your other tests are passing because of this.